### ISSUE TITLE
[BUG] pilerimport at v1.4.7 release doesn't honor -f switch 

### ISSUE THREAD

**Issue by rabsym** [jsuto/piler#218]
Hello,

After updating to 1.4.7 version looks like pilerimport doesn't honor -f switch when importing from IMAP. Sorry if my mistake again but tried everything. Maybe also curl related?

Attached you have screen captures with outcomes from 1.4.7 & 1.4.6 versions.

Thanks

![image](https://github.com/user-attachments/assets/aaafc69c-68fc-4f96-866f-b885ca5f5a6c)

![image](https://github.com/user-attachments/assets/8ccf2a6f-f5b0-478d-82c8-572fcff9cf78)

**Comment by jsuto** [comment]
No, it's a missing feature using curl lib. The mentioned PR is supposed to fix it.
